soap/services.py

```
@@ -0,0 +1,87 @@
+        assessment = {
+            'diagnoses': []
+        }
+        plan = {
+            'medications': [],
+            'lab_orders': [],
+            'follow_up': ''
+        }
+
+        soap_report = {
+            'encounter_id': context.get('encounter_id', ''),
+            'generated_at': datetime.now().isoformat(),
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)